packages/editor-ui/src/api/cloudPlans.ts

```diff
@@ -1,11 +1,8 @@
 import type { Cloud, IRestApiContext, InstanceUsage } from '@/Interface';
 import { get } from '@/utils';
 
-export async function getCurrentPlan(
-	context: IRestApiContext,
-	cloudUserId: string,
-): Promise<Cloud.PlanData> {
-	return get(context.baseUrl, `/user/${cloudUserId}/plan`);
+export async function getCurrentPlan(context: IRestApiContext): Promise<Cloud.PlanData> {
+	return get(context.baseUrl, '/rest/admin/cloud-plan');
```

Comment:
The context.baseUrl already includes the `/rest` path, because you are passing to the function the rootStore.getRestApiContext

Reply:
Also, let's wait to merge this until the backend PR is released. Otherwise, if we decide to restart the opt-in trial, we will have a problem!

Reply:
> The context.baseUrl already includes the /rest path, because you are passing to the function the rootStore.getRestApiContext

Thanks! Added this last minute.

> Also, let's wait to merge this until the backend PR is released. Otherwise, if we decide to restart the opt-in trial, we will have a problem!

Yup, planning to wait until then, but both related PRs should be reviewed together.